### ISSUE TITLE
Crash when calculating

### ISSUE THREAD

**Issue by YaManicKill** [wwwwwwzx/3DSRNGTool#6]
Occaisionally a get a crash when clicking calculate.

On Linux, wine 2.8, using 0.7.0.

Crash dump:

```
Zeus Downloads $ wine 3DSRNGTool.exe 
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x17f7fc,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1866f4,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x18d5c4,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x194494,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x19b364,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1a2234,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1a9104,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1affd4,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1b6ea4,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1bdd74,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1c4c44,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1cbb14,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1d29e4,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1d98b4,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1e0784,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1e7654,0x176698): stub
fixme:wincodecs:JpegDecoder_Frame_CopyPalette (0x1ee524,0x176698): stub
fixme:clipboard:import_text unsupported TEXT type "TEXT"
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
Parameter name: y
  at System.Windows.Forms.DataGridView.MoveCurrentCell (System.Int32 x, System.Int32 y, System.Boolean select, System.Boolean isControl, System.Boolean isShift, System.Boolean scroll) [0x00046] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.DataGridView.OnListPositionChanged (System.Object sender, System.EventArgs args) [0x0004c] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.DataGridView.DoBinding () [0x001ba] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.DataGridView.ReBind () [0x0001b] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.DataGridView.set_DataSource (System.Object value) [0x00078] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at (wrapper remoting-invoke-with-check) System.Windows.Forms.DataGridView:set_DataSource (object)
  at Pk3DSRNGTool.MainForm.AdjustDGVColumns () [0x003af] in <d7895d287fbe47f28c31e9921bba12a4>:0 
  at Pk3DSRNGTool.MainForm.Search_Click (System.Object sender, System.EventArgs e) [0x0023d] in <d7895d287fbe47f28c31e9921bba12a4>:0 
  at System.Windows.Forms.Control.OnClick (System.EventArgs e) [0x00019] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.Button.OnClick (System.EventArgs e) [0x0001e] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.ButtonBase.OnMouseUp (System.Windows.Forms.MouseEventArgs mevent) [0x00069] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.Button.OnMouseUp (System.Windows.Forms.MouseEventArgs mevent) [0x00000] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.Control.WmLButtonUp (System.Windows.Forms.Message& m) [0x00078] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.Control.WndProc (System.Windows.Forms.Message& m) [0x001b4] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.ButtonBase.WndProc (System.Windows.Forms.Message& m) [0x00037] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.Button.WndProc (System.Windows.Forms.Message& m) [0x00000] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.Control+ControlWindowTarget.OnMessage (System.Windows.Forms.Message& m) [0x00000] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.Control+ControlNativeWindow.WndProc (System.Windows.Forms.Message& m) [0x0000b] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
  at System.Windows.Forms.NativeWindow.WndProc (System.IntPtr hWnd, System.Windows.Forms.Msg msg, System.IntPtr wParam, System.IntPtr lParam) [0x00085] in <a0b7a37f0f114bf88aaa616f3b6409d9>:0 
```

**Comment by wwwwwwzx** [comment]
should be fixed on #912bc24
Try the nightly from [Appveyor](https://ci.appveyor.com/project/wwwwwwzx/3dsrngtool/branch/master)

**Comment by YaManicKill** [comment]
I've been using the nightly for a bit and not come across a crash since, so it looks like it's fixed. Will let you know if I come across it again (it wasn't something I could reliably reproduce). Thanks :-)
